.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -0,0 +1,21 @@
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes #
+
+#### Checklist
+
+
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added necessary documentation (if appropriate)
+
+
+
+**Changes proposed in this pull request**
+-
+-
+-
+
+**Screenshots (if appropriate)** 
+
+**Link to live demo (if appropriate):** 
+
+**Closes #**
```

Comment:
I don't this is needed if you have `Fixes #` above

Reply:
It is for the case if issue has been divided into sub issues and particular changes are done .